src/reducers/search.js

```diff
@@ -8,24 +8,23 @@ import {
   SEARCH_CLEAR,
 } from '../actions/search';
 
+let namespace;
 let loadedEntries;
 let response;
 let page;
 let searchTerm;
 
-const defaultState = Map({ isFetching: false, term: null, page: 0, entryIds: List([]), queryHits: Map({}) });
+const defaultState = Map({ isFetching: Map({}), page: 0, entryIds: List([]), queryHits: Map({}) });
```

Comment:
`isFetching` is a pretty inaccurate name at this point, let's move to something like `queries`. The shape could look like:
```
{
  search: { ... }
  [controlID]: { isFetching, term, hits }
  [controlID]: { isFetching, term, hits }
  ...
}
```

This change has implications for the rest of the implementation as well.